documentation/guides/migrating-from-v10-to-v11.md

```diff
@@ -9,32 +9,34 @@ Polaris v11.0.0 ([full release notes](https://github.com/Shopify/polaris/release
 - [Webpack support](#webpack-support)
 - [TypeScript](#type-script)
 - [Components](#components)
-  - [Removed Collapsible deprecated preventMeasuringOnChildrenUpdate prop](#removed-collapsible-deprecated-prevent-measuring-on-children-update-prop)
-  - [Removed Page deprecated breadcrumbs prop](#removed-page-deprecated-breadcrumbs-prop)
-  - [Removed Breadcrumbs deprecated breadcrumbs prop](#removed-breadcrumbs-deprecated-breadcrumbs-prop)
-  - [Removed KonamiCode](#removed-konami-code)
-  - [Removed DisplayText](#removed-display-text)
-  - [Removed Heading](#removed-heading)
-  - [Removed Subheading](#removed-subheading)
-  - [Removed Caption](#removed-caption)
-  - [Removed TextStyle](#removed-text-style)
-  - [Removed VisuallyHidden](#removed-visually-hidden)
-  - [Migrated Stack to LegacyStack](#migrated-stack-to-legacy-stack)
-  - [Migrated Card to LegacyCard](#migrated-card-to-legacy-card)
-  - [Migrated Filters to LegacyFilters](#migrated-filters-to-legacy-filters)
-  - [Migrated Tabs to LegacyTabs](#migrated-tabs-to-legacy-tabs)
-  - [Renamed Inline](#renamed-inline)
-  - [Renamed AlphaStack](#renamed-alpha-stack)
-  - [Renamed AlphaCard](#renamed-alpha-card)
-  - [Renamed AlphaFilters](#renamed-alpha-filters)
-  - [Renamed AlphaTabs](#renamed-alpha-tabs)
+  - [Removed `Collapsible` deprecated `preventMeasuringOnChildrenUpdate` prop](#removed-collapsible-deprecated-prevent-measuring-on-children-update-prop)
```

Comment:
```suggestion
  - [Removed `Collapsible` deprecated `preventMeasuringOnChildrenUpdate` prop](#removed-collapsible-deprecated-preventmeasuringonchildrenupdate-prop)
```

Reply:
Thanks so much for catching this Lo!! ❤️